main/pom.xml

```diff
@@ -55,6 +55,9 @@
                         <overlay>
                             <groupId>org.geonetwork-opensource</groupId>
                             <artifactId>web-app</artifactId>
+                            <excludes>
```

Comment:
Why are they excluded. Dont they need to be included

Reply:
This excludes the default log4j-*.xml files from the geonetwork core build.  Then we add our own files.

Reply:
When are they added?

Reply:
The build will include any files in the main directory. It adds in the org.geonetwork-opensource excluding the excludes and also adds the extension.  All through the magic of maven.

Reply:
The magic works so I'm happy with it. I will press the button when I think @jonescc has lost interest

Reply:
Sorry was mulling this over frokm last time I looked at it.

the log4j.xml file shows in GN3 settings as PROD 

The reason why we excluded all the packaged files was so we couldn't select them accidently and turn vocab logging off.

Was wondering whether we should just replace the files with ours so it looks good in settings and we always have our logging turned on

Reply:
We couldn't remove log4j.xml as its required during startup

Reply:
No liking the restrictions GN3 is placing on us

Reply:
Anyway thats my only comment - otherwise it works fine.